src/utils.c

```diff
@@ -123,6 +126,34 @@ app_get_object (GApplication *app, const gchar *name, GError **error)
           pspec = g_object_class_find_property (G_OBJECT_GET_CLASS (objval),
                                                 tokens[i]);
 
+          /*
+           * Check if we are trying to access a JS object
+           *
+           */
+          if (pspec == NULL && webview_type &&
+              g_strcmp0 (tokens[i], "JSContext") == 0 &&
+              G_TYPE_CHECK_INSTANCE_TYPE ((objval), webview_type))
+            {
+              const gchar *js_object_name = tokens[i+1];
```

Comment:
Maybe I am missing something, but this seems like the `js_object_name` may at some point be `NULL`, which the logic doesn't seem to want (with the `return_val_if_fail`).

Reply:
if its null it means call did not specify a object name after webview.JSContext which is a syntax error.